### ISSUE TITLE
Menus do not align with High Contrast 

### ISSUE THREAD

**Issue by pSatishC** [Microsoft/vscode#9878]
- VSCode Version: Code 1.3.1 (e6b4afa53e9c0f54edef1673de9001e9f0f547ae, 2016-07-12T13:35:06.227Z)
- OS Version: Windows_NT ia32 6.1.7601

Steps to Reproduce:

1. Switch to High Contrast theme in Windows
2. Open VSCode and check any menu (File, Edit, View, ...)

The menu item text is not visible as shown below
![invisible-menu-text-highcontrast-win](https://cloud.githubusercontent.com/assets/20694372/17199691/6383868c-549c-11e6-9712-ab5ac1e94aa5.png)

However hovering the mouse over a menu item will make the text visible since the selection background is blue. The key bindings are also visible without mouse hover since the text color is gray.
The menu decoration should consider aligning the background and text color with High Contrast settings.

**Comment by kieferrm** [comment]
Window's high contrast theme colors for background, text, and selected text are ignored by the chromium menus.

![10gnrcxcih](https://cloud.githubusercontent.com/assets/4674940/17226567/f84e889e-54be-11e6-8ff2-2a2952d6b01f.gif)

**Comment by bpasero** [comment]
This should be reported against Electron, nothing we control.

**Comment by bpasero** [comment]
https://github.com/Microsoft/vscode/issues/6629
